model-sync-gradle-test/src/test/kotlin/org/modelix/model/sync/gradle/test/PullTest.kt

```diff
@@ -0,0 +1,76 @@
+package org.modelix.model.sync.gradle.test
+
+import org.junit.jupiter.api.Test
+import java.io.File
+import kotlin.test.assertEquals
+
+class PullTest {
+
+    @Test
+    fun `nodes were synced to local`() {
+        val localModelFile = File("build/test-repo/solutions/GraphSolution/models/GraphSolution.example.mps")
+        val expected = """
```

Comment:
Maybe it would be better to put this into a file that's being read from resources. That way, one would actually have editor support for reading the long-ish XML file.